pyproject.toml

```diff
@@ -1,11 +1,26 @@
 [tool.poetry]
 name = "logdetective"
 version = "0.1.0"
-description = ""
+description = "Log using LLM AI to search for build/test failures and provide ideas for fixing these."
 authors = ["Jiri Podivin <jpodivin@gmail.com>"]
 license = "Apache-2.0"
 readme = "README.md"
 include = ["logdetective/drain3.ini"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development :: Debuggers",
+    "Topic :: Internet :: Log Analysis"
+]
+
+[tool.poetry.urls]
+Homepage = "https://github.com/fedora-copr/logdetective"
```

Comment:
I believe these should be in lower case. https://python-poetry.org/docs/pyproject/#urls

Reply:
I have it from some example but I don't think it's correct so - Fixed.